src/components/validationStatus.vue

```diff
@@ -125,21 +128,21 @@ $warning-color: #F0AD4E;
   }
 
   &__list {
-    display: flex;
```

Comment:
We want to use flex. I recommend we investigate updating the layout to break the viewport into "panes", much like many of our tools, we use flexbox with bootstrap utility classes ideally.